taxi/urls.py

```diff
@@ -39,13 +43,34 @@
     ),
     path("cars/", CarListView.as_view(), name="car-list"),
     path("cars/<int:pk>/", CarDetailView.as_view(), name="car-detail"),
+    path(
+        "car/<int:pk>/assign/",
+        views.assign_driver,
+        name="car-assign-driver"
+    ),
+    path(
+        "car/<int:pk>/remove/",
+        views.remove_driver,
+        name="car-remove-driver"
```

Comment:
Similar to the previous comment, consider using 'cars' instead of 'car' in the URL path for consistency with other car-related paths.